### PR TITLE
Replace kinto_signer by kinto.signer in settings.

### PR DESCRIPTION
**Pull request by almet**
It is done accordingly in other plugins.

r? @Natim @leplatrem

### CODE REVIEW

File: kinto_signer/hook.py [hunk 1]
```diff
@@ -11,12 +11,12 @@ def includeme(config):
     # signer.
     default_signer_module = "kinto_signer.signer.local_ecdsa"
     signer_dotted_location = settings.get(
-        'kinto_signer.signer', default_signer_module)
+        'kinto.signer.signer_backend', default_signer_module)
```
Comment (Natim) [inline kinto_signer/hook.py]: Then you probably want to get ``settings.get('signer.signer_backend')``

File: kinto_signer/hook.py [hunk 2]
```diff
@@ -11,12 +11,12 @@ def includeme(config):
...
     signer_module = config.maybe_dotted(signer_dotted_location)
     config.registry.signer = signer_module.load_from_settings(settings)
 
     # Check source and destination resources are configured.
-    raw_resources = settings.get('kinto_signer.resources')
+    raw_resources = settings.get('kinto.signer.resources')
```
Comment (Natim) [inline kinto_signer/hook.py]: Same here.

File: kinto_signer/hook.py (hunk 1 above)
Comment (almet) [inline kinto_signer/hook.py]: you mean that I don't need to specify the "kinto" prefix in this case?

**Comment by almet** [comment]
Updated.

File: kinto_signer/signer/autograph.py [hunk 3]
```diff
@@ -26,6 +26,6 @@ def sign(self, payload):
 
 def load_from_settings(settings):
     return AutographSigner(
-        server_url=settings['kinto_signer.autograph.server_url'],
-        hawk_id=settings['kinto_signer.autograph.hawk_id'],
-        hawk_secret=settings['kinto_signer.autograph.hawk_secret'])
+        server_url=settings['kinto.signer.autograph.server_url'],
```
Comment (Natim) [inline kinto_signer/signer/autograph.py]: Here too.

File: kinto_signer/signer/local_ecdsa.py [hunk 4]
```diff
@@ -90,11 +90,11 @@ def verify(self, payload, signature_bundle):
 
 
 def load_from_settings(settings):
-    private_key = settings.get('kinto_signer.ecdsa.private_key')
-    public_key = settings.get('kinto_signer.ecdsa.public_key')
+    private_key = settings.get('kinto.signer.ecdsa.private_key')
```
Comment (Natim) [inline kinto_signer/signer/local_ecdsa.py]: here too.

**Pull request merged**